frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTableConfig.tsx

```diff
@@ -198,6 +210,38 @@ const generateTableHeaders = ({
         />
       ),
     },
+    {
+      title: "Automations",
+      Header: "Automations",
+      disableSortBy: true,
+      accessor: "automations_enabled",
+      Cell: (cellProps: IStringCellProps): JSX.Element => {
+        let status;
+        if (cellProps.cell.value) {
+          if (cellProps.row.original.interval === 0) {
+            status = "paused";
```

Comment:
@rachaelshaw do we want to use a tooltip indicator (e.g. grey text or the dotted underline) so the user knows there's a tooltip on Paused?

https://www.figma.com/file/hdALBDsrti77QuDNSzLdkx/%F0%9F%9A%A7-Fleet-EE-(dev-ready%2C-scratchpad)?type=design&node-id=17610-219601&mode=design&t=YxhKlXzjDI6LCsqv-0

Reply:
I haven't seen the tooltip indicator in other table cells with tooltips, only the headings, so let's go without. (The performance impact empty state doesn't have the dotted line but has a tooltip, for example.)